app/src/main/java/com/github/se/eventradar/ui/login/LoginScreen.kt

```diff
@@ -33,17 +46,35 @@ import com.github.se.eventradar.ui.navigation.NavigationActions
 import com.github.se.eventradar.ui.navigation.Route
 
 @Composable
-fun LoginScreen(navigationActions: NavigationActions) {
+fun ErrorDialogBox(openErrorDialog: MutableState<Boolean>) {
+  val display by openErrorDialog
+  if (display == true) {
+    AlertDialog(
+        icon = { Icon(Icons.Default.AccountCircle, contentDescription = "Account Icon") },
+        text = { Text("Sign in Failed. Please try again.", textAlign = TextAlign.Center) },
+        title = { Text("Sign in Failed") },
+        onDismissRequest = { openErrorDialog.value = false },
+        confirmButton = {
+          TextButton(onClick = { openErrorDialog.value = false }) { Text("Confirm") }
```

Comment:
nitpick: I'm thinking "Ok" instead of "Confirm" makes more sense? We didn't have a Figma mockup for this, but it's a small thing so no need for one.

Reply:
👍